web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -57,6 +57,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                 },
                 getIconUrl: function(values) {
                     return Ext.ux.Ajax.constructProxyUrl(this.getGeonetworkImageUrl(values.iconSourceUuid));
+                },
+                getTrimmedTitle: function(values) {
+                    var title = values.title;
+                    return String.format("<h3 title=\"{0}\">{1}</h3>", title, Ext.util.Format.ellipsis(title, 180, true));
```

Comment:
Do you think it will be better if we return just `Ext.util.Format.ellipsis(title, 180, true)` and update the call `{[this.getTrimmedTitle(values)]}` to `<h3 title="{[values.title]}">{[this.getTrimmedTitle(values)]}</h3>`  

Something like this
```
getTrimmedTitle: function(title) {
                    return Ext.util.Format.ellipsis(title, 180, true);
                  }
```